src/test/java/org/osiam/client/OsiamUserServiceTest.java

```diff
@@ -105,6 +104,52 @@ public void invalid_filter_generates_bad_request() {
         service.searchUsers(new QueryBuilder().filter(filter).build(), accessToken);
     }
 
+    @Test
+    public void passing_attributes_to_a_request_for_all_users_generates_the_correct_request() {
```

Comment:
You are right, codacy is wrong here. However I don't see any way to make the adjustments you asked for with mockserver. I see your point (and codacy's) but I think they are mitigated  by the fact that mockserver is a tool one needs to know at least a little bit about to understand the test. As soon as they do that the test becomes obvious.

That being said, I know it's unfortunate that the syntax diverges from the common style, but I couldn't find a way to fix that.